src/client/components/LoginForm/Validation.js

```diff
@@ -0,0 +1,21 @@
+function Validation(values) {
+  const errors = {};
+  if (!values.email) {
+    errors.email = 'Email is required';
+  } else if (
+    !/^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/.test(
```

Comment:
normally if you have a input type="email" it gives a message if we missed a @ I dont understand this regular expression.

Reply:
The input type = "email" checks only for the @ symbol. but this expression is used to check all the characters like numbers, specialcharacters